pi-commerce.md

```diff
@@ -33,6 +33,7 @@ Copy the format below, add it to the bottom of the table, and update it with you
 | Local Commerce Hub | A commerce app project based on the Pi CT mockups | Typescript/React, Node.js/Express, MongoDB | [local-commerce-hub](https://github.com/pi-apps/local-commerce-hub)| N/A | N/A | N/A |
 | EasyGoods | A commerce project for pi commerce hackathon | Javascript/NextJS, FireStore | [EasyGoods](https://github.com/0205miss/EasyGoods) | [EasyGoods](https://easygoods5604.pinet.com) |
 | Map Of Pi | Map of Pi is a mobile application developed to help Pi community members easily locate local and global businesses that accept Pi as payment | TypeScript/Angular, NodeJs/Express/MongoDB | [map-of-pi](https://github.com/swoocn/map-of-pi) | [Map Of Pi](https://mapofpi9975.pinet.com) |
+| PyNook | PyNooK is a global barter mall. Through positioning, it can help pi pioneers quickly find merchants who accept pi payment, so as to achieve online and offline fast barter | Javascript/NextJS | [PyNook](https://github.com/picnty/pynook) | [PyNook](https://pynook7269.pinet.com) |
```

Comment:
| PyNook | PyNooK is a global barter mall. Through positioning, it can help pi pioneers quickly find merchants who accept pi payment, so as to achieve online and offline fast barter | Javascript/NextJS | [PyNook](https://github.com/picnty/pynook) | [PyNook](https://pynook7269.pinet.com) |